templates/flavors/kubeadm/konnectivity/allow-konnectivity-port.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ${CLUSTER_NAME}-konnectivity-cilium-policy
+data:
+  cilium-policy.yaml: |-
+    apiVersion: "cilium.io/v2"
+    kind: CiliumClusterwideNetworkPolicy
+    metadata:
+      name: "allow-konnectivity-policy"
+    spec:
+      description: "allow incoming connections to konnectivity-server from everywhere"
+      nodeSelector: {}
+      ingress:
+        - fromEntities:
+            - world
+          toPorts:
+            - ports:
+                - port: "6443"
+                - port: "8132"
```

Comment:
We have seen that sometimes konnectivity-cilium-policy gets applied first than the default cilium policies and port 6443 gets blocked (default behavior of cilium to block ports if any ccnp is present). There is no ordering of different clusterresourceset when applied. Due to this, cluster installs get stuck if konnectivity ccnp is applied as one cannot reach to cluster over 6443 port. Hence, here we are explicitly allowing 6443 here so that even if konnectivity policy gets applied first, we still have 6443 port open. I am open to discussions if we want to take a different approach.

Reply:
Updated to use patch instead.